crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -618,25 +627,39 @@ en:
           everything you need to use OpenStack APIs.
       backup:
         continue: 'Continue Upgrade Process'
-        header: 'Data Backup'
+        blockui: 'Continue Upgrade Process...'
+        header: 'Database Dump'
+        download: |
+          You can download the database dump from your node <b>%{node}</b>, you can find the file at
+          <b>%{path}</b>.
         description: |
           The SQL dump of the database was created. This will be reimported later when reapplying
```

Comment:
I know it's not from this PR, but can you change reapplying to re-applying? Same for re-imported.